### PR TITLE
Add R package mrgsolve to conda-forge

### PR DESCRIPTION
**Pull request by apeltzer**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

THere is no separate LICENCE file but instead they hold a copy of GPLv2 in each file.

### CODE REVIEW

**Comment by dbast** [comment]
Hi @apeltzer do yourself a favour and use https://github.com/bgruening/conda_r_skeleton_helper to generate the recipe... requires less manual changes on top.

**Comment by apeltzer** [comment]
Anyone can have a look at ping @conda-forge/help-r ?

File: recipes/r-mrgsolve/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    - $R -e "library('mrgsolve')"           # [not win]
+    - "\"%R%\" -e \"library('mrgsolve')\""  # [win]
+
+
+about:
+  home: https://github.com/metrumresearchgroup/mrgsolve
+  license: GPL-2.0-or-later
+  summary: Simulate from ODE-based population PK/PD and QSP models in R
+  license_family: GPL
+  license_file: mrgsolve_licence.txt
```
Comment (dbast) [inline recipes/r-mrgsolve/meta.yaml:52]: No need to contain an extra file here... r-base already provides most of the licenses used by cran packages.
```suggestion
  license_file:
    - {{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2
```

File: recipes/example/meta.yaml [hunk 2]
```diff
@@ -86,4 +86,4 @@ extra:
     # GitHub IDs for maintainers of the recipe.
     # Always check with the people listed below if they are OK becoming maintainers of the recipe. (There will be spam!)
     - LisaSimpson
-    - LandoCalrissian
+    - LandoCalrissian
```
Comment (dbast) [inline recipes/example/meta.yaml:89]: don't touch the example

**Comment by dbast** [comment]
@apeltzer see my comment before and the inline comments... this pr is also missing the build.sh and bld.bat files... regenerating via https://github.com/bgruening/conda_r_skeleton_helper is the easiest ...

**Comment by apeltzer** [comment]
Better approach now: https://github.com/conda-forge/staged-recipes/pull/15508

**Pull request closed without merging**